### ISSUE TITLE
amd64/debian:bullseye: E: Could not configure 'libc6:i386'. 

### ISSUE THREAD

**Issue by pasrom** [debuerreotype/docker-debian-artifacts#109]
Hello,

I'm using the latest docker image (output of `docker inspect amd64/debian:bullseye` can be found [here](https://pastebin.com/007P457P)).
```
docker run -it amd64/debian:bullseye
```

Inside the container I run these commands to add the i386 architecture:
```
dpkg --add-architecture i386 && apt update
apt install libc6-dev:i386
```
The last command (`apt install libc6-dev:i386`) throws following errors ([full output](https://pastebin.com/uE4S7prC)):
```
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libcrypt1:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libcom-err2:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libkrb5support0:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libk5crypto3:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libkeyutils1:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libssl1.1:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libkrb5-3:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libgssapi-krb5-2:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libtirpc3:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libnsl2:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
E: Could not configure 'libc6:i386'. 
W: Could not perform immediate configuration on 'libnss-nis:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
```

When I'm doing an `apt update && apt upgrade` everything works. Can someone help me without having to do `apt upgrade`?

many thanks and regards
pasrom

**Comment by tianon** [comment]
This sounds like a case of https://bugs.launchpad.net/bugs/1871268, which is fixed in `apt` version 2.1.12 (which hasn't quite migrated to bullseye yet):

```
apt (2.1.12) unstable; urgency=medium

  [ Julian Andres Klode ]
...
  * Do not immediately configure m-a: same packages in lockstep (LP: #1871268)
```

On the other hand, there's a Debian release this weekend (and an OpenSSL security release dropping next Tuesday or so), so there is an image rebuild imminent which should fix the situation that causes this (even if APT 2.1.12 doesn't migrate in time for it).

**Comment by pasrom** [comment]
Hello @tianon,

thank you for the quick reply and for explaining me why it doesn't work yet.
So I'll wait until next week :)

regards
pasrom
